.textlintrc.json

```diff
@@ -15,7 +15,10 @@
         "/<http://.*>/",
         "medium.com/opentelemetry",
         "github.com/open-telemetry",
-        "#opentelemetry"
+        "#opentelemetry",
+        "/  - .*/",
+        "/(?:language|repo): .*/",
+        "net/http"
```

Comment:
Is this entry still necessary given that you've put occurrences of "net/http" inside code fences, like this:

    `net/http`

Reply:
There is at least one instance were net/http is in the title of a registry entry, where we do not apply `markdownify`, that's why I put it here. I need to raise an issue with the registry because of that, but we can keep it out for now

Reply:
Right, I'd still apply code formatting, and drop this entry. But, in the end, it's your call.

Reply:
```suggestion
```